tests/integration_tests/tests/agentless_tests/test_rest_service_sort.py

```diff
@@ -50,8 +50,19 @@ def test_executions_sort(self):
         for i in range(5):
             self.execute_workflow('install', deployment.id)
             self.execute_workflow('uninstall', deployment.id)
-        self._test_sort('executions',
-                        ['deployment_id', '-status'])
+
+        api = getattr(self.client, 'executions')
+        sorted_list = api.list(_sort=['-workflow_id', 'status'])
+        self.assertGreater(len(sorted_list), 0)
```

Comment:
If you expecting the list to have at least 11 items, then perhaps change the assertion to at least 11..